src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -130,10 +129,20 @@ afterEach(() => {
   jest.restoreAllMocks();
 });
 
-test('renders coming occurrences table correctly', async () => {
-  initializeMocks();
-  renderWithRoute(<CreateOccurrencePage />, {
+const InitializeMocksAndRenderPage = (
```

Comment:
Is the capital letter at start intentional? :)

Reply:
fixed